packages/contracts/contracts/interfaces/v0.8.x/ISharedMinterDAV0.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface ISharedMinterDAV0 {
+    /// Auction details cleared for project `projectId`.
+    event ResetAuctionDetails(
+        uint256 indexed _projectId,
+        address indexed _coreContract
+    );
+
+    function projectAuctionParameters(
```

Comment:
nit: can you add a smol dev comment explaining why the return values are not given names here (because specific minter implementations return different values in each slot)?

Additionally, I think we might want to return `uint64` and `uint128` values here?